Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -350,7 +350,7 @@ void RenderBox::styleDidChange(StyleDifference diff, const RenderStyle* oldStyle
     // If our zoom factor changes and we have a defined scrollLeft/Top, we need to adjust that value into the
     // new zoomed coordinate space.
     if (hasNonVisibleOverflow() && layer() && oldStyle && oldStyle->usedZoom() != newStyle.usedZoom()) {
-        if (auto* scrollableArea = layer()->scrollableArea()) {
+        if (CheckedPtr<RenderLayerScrollableArea> scrollableArea = layer()->scrollableArea()) {
```

Comment:
```suggestion
        if (CheckedPtr scrollableArea = layer()->scrollableArea()) {
```
(same goes for all other uses of CheckedPtr in this PR)

Reply:
Thanks for the suggestion. I’ve updated the code to use CTAD (CheckedPtr scrollableArea = …).
This keeps the declaration concise and lets the pointee type (including constness) be deduced from the initializer.

Reply:
Perfect!

Reply:
@lilyspiniolas 
Thank you for your time reviewing this patch.  
From what I can tell, the Windows EWS failure doesn’t seem related to the changes in this patch, and may be due to an infrastructure issue.  
Would you be able to re-trigger the Windows EWS, or proceed with landing if appropriate?  
I’d greatly appreciate your assistance.

Reply:
Of course, thank you for your contribution! The Windows EWS failure seems unrelated to your change, so I will go ahead and land this.

Reply:
@lilyspiniolas  Thank you so much for taking the time to review my PR and helping me get it landed! This is my first contribution to WebKit and your guidance made it possible. I really appreciate your patience and detailed feedback throughout the process. Looking forward to contributing more! 🙏